src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1046,6 +1046,10 @@ private function addMessengerSection(ArrayNodeDefinition $rootNode)
                         ->ifTrue(static function ($v): bool { return isset($v['buses']) && null !== $v['default_bus'] && !isset($v['buses'][$v['default_bus']]); })
                         ->then(static function (array $v): void { throw new InvalidConfigurationException(sprintf('The specified default bus "%s" is not configured. Available buses are "%s".', $v['default_bus'], implode('", "', array_keys($v['buses'])))); })
                     ->end()
+                    ->booleanNode('auto_routing')
+                        ->info('Automatically create routing for handlers that specify "from_transport" via tag attribute or MessageSubscriberInterface.')
+                        ->defaultFalse()
```

Comment:
I think this should be `defaultTrue()` in Symfony 6. But that is a separate discussion.

Reply:
How and where do you keep track of this?

Reply:
I am not sure. I wanted to write a note here to help other reviewers to pitch in. They will either say to open second PR/issue after this get merged so we can discuss if this is a good default or not, OR they will ask you to do something clever and add deprecation messages somewhere.